.travis.yml

```
@@ -274,6 +274,12 @@ matrix:
         - ! git grep -q --ignore-case "gnu general public";
         - ! git grep -q --ignore-case "gnu library general public";
 
+    - env:
+        - NAME=include_check
+      script:
+        - echo 'Checking that there are no '#include "mbed.h"' in code where it should not be'
+        - ! git grep '^#include "mbed.h"$' -- '*.c' '*.h' '*.cpp' '*.hpp' ':!*platform_mbed.h' ':!*TESTS/*' ':!TEST_APPS/' ':!UNITTESTS/' ':!*tests/*' ':!*targets/*' ':!*TARGET_*' ':!*unsupported/*'
```

Makes the line a bit more readable, imo.

```suggestion
        - >-
         ! git grep '^#include "mbed.h"$' -- '*.c' '*.h' '*.cpp' '*.hpp' \ 
             ':!*platform_mbed.h' ':!*TESTS/*' ':!TEST_APPS/' ':!UNITTESTS/' \
             ':!*tests/*' ':!*targets/*' ':!*TARGET_*' ':!*unsupported/*'
```